Topic7/src/main/java/com/aguirre/topic7/controllers/CourseController.java

```diff
@@ -0,0 +1,48 @@
+package com.aguirre.topic7.controllers;
+
+import java.util.List;
+
+import javax.persistence.Entity;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import com.aguirre.topic7.models.Course;
+import com.aguirre.topic7.models.CourseDao;
+import com.aguirre.topic7.models.Student;
+import com.aguirre.topic7.models.StudentCourseDao;
+import com.google.gson.Gson;
+
+@Entity
+@RequestMapping("/courses")
+public class CourseController {
+
+	@Autowired
+	private CourseDao courseDao;
+	private StudentCourseDao studentCourseDao;

```

Comment:
You should set an @autowired annotation in studentCourseDao too to inject the StudentCourseDao